js/src/reports/format-amount.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { numberFormat } from '@woocommerce/number';
+
+/**
+ * Internal dependencies
+ */
+import formatAmountWithCode from '.~/utils/formatAmountWithCode';
+
+const unavailable = __( 'Unavailable', 'google-listings-and-ads' );
+
+/**
+ * Formats given number as currency
+ * according to given config, or return `"Unavailable"` if the value is undefined.
+ *
+ * Entire usage could be simplified once
+ * https://github.com/woocommerce/woocommerce-admin/pull/7575 is released and accessible.
+ *
+ * @param {number} value Value to be formatted.
+ * @param {Object} currencyConfig Currency config object returned by `@woocommerce/currency`.
+ * @return {string} Formatted currency or "Unavailable".
+ */
+function formatCurrencyCell( value, currencyConfig ) {
+	if ( value === undefined ) {
+		return unavailable;
+	}
+	return formatAmountWithCode( currencyConfig, value );
+}
+/**
+ * Formats given number as Ads' currency
+ * or return `"Unavailable"` if the value is undefined.
+ *
+ * @param {number} value Value to be formatted.
+ * @param {Object} storeCurrencyConfig Store's currency config object returned by `@woocommerce/currency`.
+ * @param {Object} adsCurrencyConfig Ads' currency config object returned by `@woocommerce/currency`.
```

Comment:
💅 
Should the source of `adsCurrencyConfig` refer to `useAdsCurrency` hook?

Reply:
Sorry, but I don't get it. Do you mean we should refer here to `useAdsCurrency` instead of `@woocommerce/currency`?

Reply:
It might be a bit subtle to specify where the parameter should be obtained from in the JSDoc, but if it needs to, I thought it could reduce confusion to indicate that the passing currency config `adsCurrencyConfig` should be the one got from `useAdsCurrency` hook.

---

I just noticed that `@woocommerce/currency` has its own [currency config](https://github.com/woocommerce/woocommerce-admin/blob/v2.1.5/packages/currency/src/index.js#L15-L22), and this currency config is neither the store nor the ads. Probably the descriptions of `storeCurrencyConfig` need adjustments as well.